<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoboCAT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabEquips">
       <attribute name="title">
        <string>&amp;Equips</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableView" name="taulaEquips">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>120</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutEquips">
          <item>
           <widget class="QPushButton" name="afegirEquips">
            <property name="text">
             <string>&amp;Afegir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editarEquips">
            <property name="text">
             <string>&amp;Editar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="eliminarEquips">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPartides">
       <attribute name="title">
        <string>&amp;Partides</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="taulaPartides"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutPartides">
          <item>
           <widget class="QPushButton" name="afegirPartides">
            <property name="text">
             <string>&amp;Afegir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editarPartides">
            <property name="text">
             <string>&amp;Editar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="eliminarPartides">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCrono">
       <attribute name="title">
        <string>&amp;Cronòmetre</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="estatsPantallaCrono">
          <item>
           <widget class="QRadioButton" name="pantallaCronoIdle">
            <property name="text">
             <string>Idle</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">estatPantallaCronoBg</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="pantallaCronoSeguents">
            <property name="text">
             <string>Següents</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">estatPantallaCronoBg</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="pantallaCronoCronometre">
            <property name="text">
             <string>Cronòmetre</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">estatPantallaCronoBg</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pantallaCronoTaula1Layout">
          <item>
           <widget class="QCheckBox" name="pantallaCronoTaula1">
            <property name="toolTip">
             <string>Activar taula 1</string>
            </property>
            <property name="text">
             <string>Taula 1</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="pantallaCronoE1T1">
            <property name="toolTip">
             <string>Equip 1 Taula 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="pantallaCronoE2T1">
            <property name="toolTip">
             <string>Equip 2 Taula 1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pantallaCronoTaula2Layout">
          <item>
           <widget class="QCheckBox" name="pantallaCronoTaula2">
            <property name="toolTip">
             <string>Activar taula 2</string>
            </property>
            <property name="text">
             <string>Taula 2</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="pantallaCronoE1T2">
            <property name="toolTip">
             <string>Equip 1 Taula 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="pantallaCronoE2T2">
            <property name="toolTip">
             <string>Equip 2 Taula 2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ChronoLabel" name="cronometre">
          <property name="font">
           <font>
            <pointsize>75</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string notr="true">0:00</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutCrono">
          <item>
           <widget class="QPushButton" name="playCrono">
            <property name="text">
             <string>P&amp;lay</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopCrono">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetCrono">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="estatBdLabel">
        <property name="text">
         <string>Estat BD:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="estatBd">
        <property name="text">
         <string notr="true">&lt;Estat BD&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArxiu">
    <property name="title">
     <string>Ar&amp;xiu</string>
    </property>
    <addaction name="actionConnectar_a_BD"/>
    <addaction name="actionTancar_connexio"/>
    <addaction name="separator"/>
    <addaction name="actionInicialitzar_BD"/>
    <addaction name="separator"/>
    <addaction name="actionSortir"/>
   </widget>
   <widget class="QMenu" name="menuPantalles">
    <property name="title">
     <string>Pa&amp;ntalles</string>
    </property>
    <addaction name="actionPuntuacio"/>
    <addaction name="actionCronometre"/>
   </widget>
   <addaction name="menuArxiu"/>
   <addaction name="menuPantalles"/>
  </widget>
  <action name="actionSortir">
   <property name="text">
    <string>&amp;Sortir</string>
   </property>
  </action>
  <action name="actionConnectar_a_BD">
   <property name="text">
    <string>&amp;Connectar a BD</string>
   </property>
  </action>
  <action name="actionTancar_connexio">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Tancar connexió</string>
   </property>
  </action>
  <action name="actionPuntuacio">
   <property name="text">
    <string>&amp;Puntuació</string>
   </property>
  </action>
  <action name="actionCronometre">
   <property name="text">
    <string>&amp;Cronòmetre</string>
   </property>
  </action>
  <action name="actionInicialitzar_BD">
   <property name="text">
    <string>Inicialitzar BD</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ChronoLabel</class>
   <extends>QLabel</extends>
   <header>chronolabel.h</header>
   <slots>
    <slot>updateTicks(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSortir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>chronoTicked(int)</signal>
   <receiver>cronometre</receiver>
   <slot>updateTicks(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playCrono</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startChrono()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopCrono</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopChrono()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetCrono</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetChrono()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInicialitzar_BD</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>obreDialegInicialitzacio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCronometre</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>obrePantallaCrono()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pantallaCronoTaula1</sender>
   <signal>toggled(bool)</signal>
   <receiver>pantallaCronoE1T1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pantallaCronoTaula1</sender>
   <signal>toggled(bool)</signal>
   <receiver>pantallaCronoE2T1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pantallaCronoTaula2</sender>
   <signal>toggled(bool)</signal>
   <receiver>pantallaCronoE1T2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pantallaCronoTaula2</sender>
   <signal>toggled(bool)</signal>
   <receiver>pantallaCronoE2T2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>afegirEquips</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>afegirEquip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editarEquips</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>modificarEquip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>chronoTicked(int)</signal>
  <slot>startChrono()</slot>
  <slot>stopChrono()</slot>
  <slot>resetChrono()</slot>
  <slot>obreDialegInicialitzacio()</slot>
  <slot>obrePantallaCrono()</slot>
  <slot>afegirEquip()</slot>
  <slot>modificarEquip()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="estatPantallaCronoBg"/>
 </buttongroups>
</ui>
